app/screens/settings/about/about.tsx

```diff
@@ -248,6 +282,21 @@ const About = ({componentId, config, license}: AboutProps) => {
                         })}
                     </Text>
                 </View>
+
+                <TouchableOpacity style={styles.copyInfoButtonContainer}>
```

Comment:
The `button` component already has a touchable. Why do you need this one?

Reply:
okay cool! but i need to use some other container for that as the Button component fillls the entire scren!

Reply:
Oh, I see where the issue might be. Let's try the following:

```
                    <Button
                        theme={theme}
                        backgroundStyle={styles.copyInfoButtonContainer}
                        onPress={copyToClipboard}
                        text={'Copy info'}
                        testID={'about.copy_info'}
                        iconName='content-copy'
                        iconSize={12}
                        buttonType={'default'}
                        size={'m'}
                        emphasis={'tertiary'}
                    />
```

The `buttonBagroundStyle` and `buttonTextStyle` are assigned automatically inside the Button component, so you just need to overwrite the needed styles (the styles you are passing to `backgroundStyle`). That should be enough, but let me know if anything seems off.

Reply:
@Sudhanva-Nadiger Did you check this comment?

Reply:
Oh sorry! I missed that comment!
I will check that and let you know what happens if we use that